### PR TITLE
changed method t to translation

### PR DESCRIPTION
**Pull request by swathi-0901**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by swathi-0901** [comment]
So can u please tell me where exactly I went wrong and what am I supposed to do?

File: app/controllers/wiki_controller.rb [hunk 1]
```diff
@@ -456,9 +456,7 @@ def update_node_attributes
         main_image.save
       end
 
-      if params[:main_image].to_i == 0
-        @node.main_image_id = nil
-      elsif params[:main_image].present? && img = Image.find(params[:main_image])
+      if params[:main_image].present? && img = Image.find(params[:main_image])
```
Comment (nstjean) [inline app/controllers/wiki_controller.rb]: Hi, looks like you deleted something here! There shouldn't be any edits beyond changing the t functions.

**Comment by nstjean** [comment]
@swathi-0901 Are you still working on this?

File: app/controllers/wiki_controller.rb (hunk 1 above)
Comment (swathi-0901) [reply app/controllers/wiki_controller.rb]: > Hi, looks like you deleted something here! There shouldn't be any edits beyond changing the t functions.

yea made the required changes in another pull request, I have sent can please review it...also can I know that the changes I made are correct or not?

**Pull request closed without merging**